packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -552,6 +553,8 @@ export const GiantTrackTile = ({
           trackId={trackId}
           coverArtSizes={coverArtSizes}
           coSign={coSign}
+          // @ts-ignore - artwork is on track, but was removed previously
+          artwork={track?.artwork?.[SquareSizes.SIZE_1000_BY_1000]}
```

Comment:
just trying to understand where this is coming from? is it added to the track metadata in the ssr part before the render? if so i think we should reconsider how the data gets passed down to avoid two separate flows for images on client vs server